src/api.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+const api = axios.create({
+  baseURL: "https://board-games-api-q1cq.onrender.com/api/",
```

Comment:
Love the setup of a base url here, makes switching hosts super easy if you ever had to 💯